komodo/symlink/create_links.py

```diff
@@ -30,6 +39,7 @@ def _create_link(src, dst, link_dict):
             return
         os.remove(dst)
         print(f"Existing symlink {dst} moved from {existing_link} to {src}")
+        locate_implicitly_moved_symlinks(dst, link_dict)
```

Comment:
It is hard to read from the function name that this has a side-effect to do a `print`. I think it should return a string that we can inject into the print statement on the line above (then we can format it inside a paranthesis e.g).

Reply:
Yes, the method should just return the name of the symlink that was moved, so we can format it later on.